pkg/graveler/ref/tag_iterator_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
-func TestTagIterator(t *testing.T) {
+func TestDBTagIterator(t *testing.T) {
```

Comment:
Let's reuse code and not duplicate it.

Reply:
I don't think we need to reuse it in this case because the iterators have different signature